content/zh-tw/become-a-sponsor.html

```diff
@@ -2,7 +2,7 @@
 title: 成為贊助商
 slug: become-a-sponsor
 top_graphic: 4
-lastmod: 2019-01-28
+lastmod: 2019-08-29
```

Comment:
If you update the lastmod you must include those changes: https://github.com/letsencrypt/website/commit/0659ac04419d6d3d3b9a48a7c3a9f6220d0f8611#diff-e2aed2a0943218bfdd63c46e50293a10

Reply:
Thank you, I forgot to change them.